quizapp/src/main/java/billennium/quizapp/service/CandidateService.java

```diff
@@ -21,4 +25,14 @@ public String findByEmail(String email) {
                 .orElseThrow(CandidateException::new);
     }
 
+    public void saveCandidate(String email) {
+        candidateRepository.save(
+                Candidate.builder()
+                        .email(email)
+                        .quizExecuted(QuizExecuted.builder()
+                                .quiz(quizDefinitionRepository.findAll().stream().findAny().orElseThrow(QuizDefinitionException::new))
```

Comment:
Tutaj przypisujesz obojetnie jaki quiż i przypisujesz go do użytkownika. tak ma byc ?

Reply:
póki co myślę że tak, przypisuje go do użytkownika bo póki co skupiamy się na funkcjonalności żeby utworzyć użytkownika przypisać mu quiz i żeby zapisysać jego wynik

Reply:
póki co tak randomowo wybrać test przypisać go użytkownikowi. 
Były jeszcze pomysły by np. dodać status quizu np busy ale jak będzie tylko kilka quizów i każdy będzie miał przypisanego użytkownika to szybko się skończą :) a np status będziemy zmieniać dopiero jak quiz się zakończy. Mamy co prawda enum QuizStatus w QuizExecuted ale ten w moim odczuciu to infoamcja dla nas że kandadat rozpoczął quiz.

Reply:
Ale napisz tam jakieś todo ze to tylko tymaczasowe rozwiązanie

Reply:
Ok dopisałem 👍